metadata-ingestion/src/datahub/ingestion/extractor/schema_util.py

```diff
@@ -314,6 +322,8 @@ def emit(self) -> Generator[SchemaField, None, None]:
                     description = (
                         f"<span style=\"color:red\">DEPRECATED: {merged_props['deprecated']}</span>\n"
                         + description
+                        if description
```

Comment:
doesn't this effectively do the same thing you are trying to prevent above?

If description is None we set it to "" ... so why not just set description to "" above

Reply:
A bunch of other integration test expects this property as None and not empty string in their golden files. (like Glue, Hive, etc...) and those broke when I had empty string.
This way the output is identical what we had before.